Website/.gitignore

```diff
@@ -1,22 +1,5 @@
-.DS_Store
 node_modules
-
-
-# local env files
-.env.local
-.env.*.local
-
-# Log files
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-
-# Editor directories and files
-.idea
-.vscode
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
+.DS_Store
+# dist
```

Comment:
Because the actual way of deployment I am not ignoring the dist folder, but it should change in future